docs-ref-conceptual/install-azure-cli.md

```diff
@@ -35,6 +35,31 @@ The Azure CLI is available to install in Windows, macOS and Linux environments.
 > [!NOTE]
 > If you're using the Azure classic deployment model, [install the Azure classic CLI](install-classic-cli.md).
 
+## FAQ
+
+### Where is the Azure CLI installed?
+
+When installing the Azure CLI, you cannot select an install location. In Windows, the 32-bit Azure CLI installs in `C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2` and the 64-bit in `C:\Program Files\Microsoft SDKs\Azure\CLI2`. In Linux, programs are more distributed.  Most binaries are in `/bin` or `/sbin` and most libraries are in `/lib`.
```

Comment:
```suggestion
When installing the Azure CLI, you cannot select an install location. In Windows, the 32-bit Azure CLI installs in `C:\Program Files (x86)\Microsoft SDKs\Azure\CLI2` and the 64-bit in `C:\Program Files\Microsoft SDKs\Azure\CLI2`. In Linux, programs are more distributed. It's located in `/opt/az/` on Ubuntu and Debian,  `/lib64/az/` on CentOS, RHEL and Azure Linux
```

Reply:
Thanks, @bebound!  I've added this information.